source/StoryMenuState.hx

```diff
@@ -72,8 +72,21 @@ class StoryMenuState extends MusicBeatState
 	var leftArrow:FlxSprite;
 	var rightArrow:FlxSprite;
 
+	function unlockWeeks():Array<Bool>
+	{
+		var weeks:Array<Bool> = [true];
+
+		for(i in 0...FlxG.save.data.weekUnlocked)
+			{
+				weeks.push(true);
+			}
```

Comment:
this code sucks, was going to say to like use the function that fills an array here but apparently that Doesn't Exist In Haxe ?!?!?!?! so i don't care, this can stay

Reply:
haxe's stupid